Libs/GLib/Interfaces/IHandle.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace GLib.Interfaces
```

Comment:
This should just be `GLib`, not `GLib.Interfaces` to be consistent.

Reply:
Done